package.json

```diff
@@ -1780,8 +1780,8 @@
         },
         "IBM i Project Explorer.projectScanDepth": {
           "type": "integer",
-          "default": 2,
-          "minimum": 1,
+          "default": 0,
+          "minimum": 0,
```

Comment:
Why are we setting the default scan depth to 0

Reply:
@SanjulaGanepola I wasn't sure why this was an improvement?

Reply:
I was proposing this change because I felt that the notification would appear too often as they would get it every time they are dealing with a project with sub projects. This could be an annoyance if they do not intend to work with sub projects.

Now that I think about it, instead of setting the default to 0, should we keep it at 2 and instead remove the automatic checking for sub projects? This would leave it up to the users to explicitly `Scan For Projects` since this is more of a niche thing that only certain people would use. What do you think?

![image](https://github.com/user-attachments/assets/e6e50fda-8bb6-4358-b5dc-25c7741232cd)